ambry-tools/src/main/java/com.github.ambry.tools/admin/MigrationTool.java

```diff
@@ -1,170 +0,0 @@
-/**
```

Comment:
removed this because it is unused and migrating it to the `Router` was unnecessary right now. There is already an issue (#383) to write a more comprehensive migrator service. I can like this PR to the issue in case someone wants to see what was available.